scripts/count_jobs.py

```diff
@@ -153,8 +142,30 @@ def main():
 
     print(f"Found a total of {len(jobs)} unique jobs across {len(commits)} commits.")
 
+    return jobs
+
+
+def main():
+    """a small helper to generate an all-time jobs count
+    and optionally produce a "master" jobs file"""
+    repository = "https://github.com/USRSE/usrse.github.io"
+    tmpdir = tempfile.mkdtemp(prefix="usrse-")
+
+    print(f"Cloning repository {repository}")
+    repo = clone_repo(repository, dest=tmpdir)
+
+    # If user provided an output file, derive path before chdir
+    outfile = None
+    if len(sys.argv) > 1:
+        outfile = os.path.abspath(sys.argv[1])
+
+    # Change directory to the repo to get list of commits
+    os.chdir(repo)
+
+    jobs = [count_jobs("_data/jobs.yml"), count_jobs("_data/related-jobs.yml")]
```

Comment:
This is going to be a list of lists - use a * to explode them into a common list.

but also this is going to be a bug to write them both to the same file - do we want that? I haven’t tested yet, just thinking out load.

Reply:
> This is going to be a list of lists - use a * to explode them into a common list.

whoops.  Fixed, I think.

> but also this is going to be a bug to write them both to the same file - do we want that? I haven’t tested yet, just thinking out loud.

Yeah, I wasn't sure about that either but I assumed that if the goal was a "grand total" list then they should both go into that file.

Reply:
yeah that would make sense! Maybe if we are combining we should add an attribute related True or related False so we can distinguish them as needed. you could add an extra param to the function related=True and then just add it to each job we save.